EventFilter/GEMRawToDigi/python/muonGEMDigis_cfi.py

```diff
@@ -1,6 +1,11 @@
 import FWCore.ParameterSet.Config as cms
 from EventFilter.GEMRawToDigi.muonGEMDigisDefault_cfi import muonGEMDigisDefault as _muonGEMDigisDefault
+from L1Trigger.L1TGEM.simMuonGEMPadDigis_cfi import simMuonGEMPadDigis as _simMuonGEMPadDigis
+from L1Trigger.L1TGEM.simMuonGEMPadDigiClusters_cfi import simMuonGEMPadDigiClusters as _simMuonGEMPadDigiClusters
+
 muonGEMDigis = _muonGEMDigisDefault.clone()
+muonGEMPadDigisReEmul = _simMuonGEMPadDigis.clone(InputCollection=cms.InputTag("muonGEMDigis"))
+muonGEMPadDigiClustersReEmul = _simMuonGEMPadDigiClusters.clone(InputCollection=cms.InputTag("muonGEMPadDigisReEmul"))
```

Comment:
```suggestion
muonGEMPadDigisReEmul = _simMuonGEMPadDigis.clone(InputCollection = "muonGEMDigis")
muonGEMPadDigiClustersReEmul = _simMuonGEMPadDigiClusters.clone(InputCollection = "muonGEMPadDigisReEmul")
```
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

Also, if these modules are to be moved to L1Reco (as proposed in the comment in RawToDigi_cff.py), then it may be better to have these definitions to stay in `L1Trigger.L1TGEM`

Reply:
OK. Can do.